internal/xds/translator/translator.go

```diff
@@ -104,7 +104,8 @@ func (t *Translator) processHTTPListenerXdsTranslation(tCtx *types.ResourceVersi
 		// Create a route config if we have not found one yet
 		if xdsRouteCfg == nil {
 			xdsRouteCfg = &routev3.RouteConfiguration{
-				Name: httpListener.Name,
+				IgnorePortInHostMatching: true,
+				Name:                     httpListener.Name,
```

Comment:
trying to better understand how the yaml already contained the `name` before this addition, is it also set somewhere else in the codebase ?

Reply:
We had `Name` set in RouteConfiguration before, we just added `IgnorePortInHostMatching` here.

Reply:
my bad 🙈